edxval/tests/test_api.py

```diff
@@ -1785,7 +1785,7 @@ def test_external_internal_transcripts_conflict(self):
         # First create external transcript in sjson format.
         en_transcript_file_name = 'external-transcript-en.sjson'
         utils.create_file_in_fs(
-            constants.VIDEO_TRANSCRIPT_CUSTOM_SJSON['file_data'],
+            self.transcript_data1['file_data'],
```

Comment:
Same for this test case.

Reply:
By the way, the change for this test case was actually a mistake. I should've changed it differently. Accordingly, I updated the docstring for this test case, and the previous one.

In addition, I also made the necessary changes to this test case so that it imports two different transcripts, an external and an internal one. And then the internal one would be overridden by the external one, because the external one gets imported after, and its contents are different.

I would appreciate it if you could give this test another look please, when you have time @DawoudSheraz :+1: